ui/app/templates/partials/secret-form-edit.hbs

```diff
@@ -4,9 +4,11 @@
     <NamespaceReminder @mode="edit" @noun="secret" />
     {{#if (not-eq model.selectedVersion.version model.currentVersion)}}
       <div class="form-section">
-        <MessageInPage @type="warning" @class="is-marginless">
-          You are creating a new version based on data from Version {{model.selectedVersion.version}}. The current version for <code>{{model.id}}</code> is Version {{model.currentVersion}}.
-        </MessageInPage>
+        <AlertBanner
+          @type="warning"
+          @class="is-marginless"
+          @message="You are creating a new version based on data from Version {{model.selectedVersion.version}}. The current version for {{model.id}} is Version {{model.currentVersion}}."
```

Comment:
OK, I take it back because it totally works 🤯 - this must be a new thing with angle bracket components, but that's awesome.